CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Snyk Changelog
 
+## [2.4.23]
+
+### Changed
+- Provide an additional info to the “Organization” setting
+
```

Comment:
we didn't release 2.4.22 yet, changes should go inder that tag for now

Reply:
done with 62bb67d788ef6d1bb09853d80af9eeb2b0f6fa5f